2018/about.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+order: 10
+title: "2018"
+subtitle: "About the Event"
+titleDisplay: "About"
+---
+
+Our Networks is a conference about the past, present, and future of building our own network infrastructures. Now in its second year, the event brings together enthusiasts, hardware and software hackers, researchers, organizers and more to collectively explore creative and critical engagements with the Internet and alternative infrastructures.
+
+### I want to make this even even better, how can I help?
+
+Awesome, we are [planning this event openly](https://github.com/ournetworks/2018)! You should consider [tabling at our Yami-ichi](/yami-ichi/) or [pitching a project for our Sprints](/2018/sprints/). If those activities don't appeal, we will need volunteers to help the event run smoothly--reach out to us at orga@ournetworks.ca!
+
+###  Organizing Team
+
+Our Networks organizers have planned workshops and hosted events on network literacy and community networks as well as built mesh networks with Toronto Mesh for the past two years:
```

Comment:
Thinking that a comma somewhere in this sentence.

Reply:
Suggestion:

>Our Networks organizers have facilitated workshops, hosted network literacy events, and built community networks with Toronto Mesh for the past two years:

Reply:
restructured